src/ziggurat/messaging/rabbitmq/cluster/producer.clj

```diff
@@ -8,12 +8,15 @@
             [clojure.string :as str])
   (:import (org.apache.kafka.common.header.internals RecordHeader)))
 
-(defn get-default-ha-policy [cluster-config]
-  (let [ha-mode      (get cluster-config :ha-mode "all")
-        ha-params    (get cluster-config :ha-params 1)
+(defn get-replica-count [host-count]
+  (int (Math/ceil (/ host-count 2))))
+
+(defn get-default-ha-policy [cluster-config host-count]
+  (let [ha-mode      (get cluster-config :ha-mode "exactly")
+        ha-params    (get cluster-config :ha-params (get-replica-count host-count))
```

Comment:
[MINOR] This is the only usage of host-count in this method. So, we could pass replica-count directly.